frontend/packages/kubevirt-plugin/integration-tests/views/vm.environment.view.ts

```diff
@@ -0,0 +1,62 @@
+import { $, $$, browser, by, ExpectedConditions as until, element } from 'protractor';
+import { addVariableFrom } from '@console/internal-integration-tests/views/environment.view';
+import {
+  PAGE_LOAD_TIMEOUT_SECS,
+  VM_BOOTUP_TIMEOUT_SECS,
+  KUBEVIRT_SCRIPTS_PATH,
+} from '../tests/utils/consts';
+import { actionsButton } from '@console/internal-integration-tests/views/crud.view';
+import { execSync } from 'child_process';
+
+export const selectResource = element(by.buttonText('Select a resource'));
+export const allPairRows = $$('.pairs-list__row');
+export const dropDownfilter = $('[data-test-id="dropdown-text-filter"]');
+export const resouceNameInput = $('.co-resource-item__resource-name');
+export const clickFirstOption = async () => {
+  await $$('[role=option]')
+    .get(0)
+    .$('.pf-c-dropdown__menu-item')
+    .click();
+};
+export const saveButton = $('.pf-c-form__actions').$('.pf-m-primary');
+export const successAlert = $('.pf-c-alert.pf-m-inline.pf-m-success.co-alert');
+export const errorAlert = $('.pf-c-alert.pf-m-inline.pf-m-danger.co-alert.co-alert--scrollable');
+
+export const startVMChoice = $('[data-test-action="Start Virtual Machine"]');
```

Comment:
Unnecessary, we hav VM actions in `packages/kubevirt-plugin/integration-tests/tests/utils/consts.ts` in VM_ACTION